code/modules/events/queen_mothers_blessing.dm

```diff
@@ -0,0 +1,37 @@
+#define QM_HEAL_AMOUNT 250
+
+/datum/round_event_control/queen_mothers_blessing
+	name = "Queen Mother's Blessing"
+	typepath = /datum/round_event/queen_mothers_blessing
+	weight = 15
+	earliest_start = 10 MINUTES
+
+	gamemode_blacklist = list("Combat Patrol","Civil War","Sensor Capture")
+
+/datum/round_event_control/queen_mothers_blessing/can_spawn_event(players_amt, gamemode)
+	if(length(GLOB.alive_xeno_list) <= 4)
+		return FALSE
+	return ..()
+
+/datum/round_event/queen_mothers_blessing/start()
+	var/sound/queen_sound = sound(get_sfx("queen"), channel = CHANNEL_ANNOUNCEMENTS)
+	for(var/mob/living/carbon/xenomorph/receiving_xeno in GLOB.alive_xeno_list)
```

Comment:
It's extremely unlikely, but it's possible this event could trigger and there's no valid xenos (i.e. shrike hijacks then dies, and it's just a bunch of larva. It may be worth having this sound part trigger in the for loop, before the bless_xeno proc gets called, to ensure it only triggers the sound if the event successfully hits a beno.